pkg/eventsapi/v1.go

```diff
@@ -40,6 +41,13 @@ func (e *EventV1) Version() EventVersion {
 // DetailsV1 corresponds to a V1 details object.
 type DetailsV1 map[string]interface{}
 
+// AgentContext is used for pdagent integrations
+type AgentContext struct {
```

Comment:
I'm bummed that we have to include this on the Event struct but there do not seem to be any quick workarounds for now.